pkg/rpc/client/rpc.go

```diff
@@ -385,18 +385,15 @@ func (c *Client) invokeSomething(method string, p request.RawParams, cosigners [
 // The given hex string needs to be signed with a keypair.
 // When the result of the response object is true, the TX has successfully
 // been broadcasted to the network.
-func (c *Client) SendRawTransaction(rawTX *transaction.Transaction) error {
+func (c *Client) SendRawTransaction(rawTX *transaction.Transaction) (*result.RelayResult, error) {
```

Comment:
Do we really need `RelayResult` wrapper? I think this API can be simplified to return Uint256 and an error.

Reply:
Removed this wrapper from RPC client, but we still need it in RPC server for proper JSON marshalling.